docs_src/design/ucr/Core-Data-Cache.md

```diff
@@ -0,0 +1,46 @@
+## Core Data Cache
```

Comment:
I don't agree that this UCR is describing [cache](https://en.wikipedia.org/wiki/Cache_(computing)) or cache-like requirements. It is true that Core Data can be used as a cache to query device data faster than if the query was sent to the device, but that is already possible. I think what is being asked here is the addition or rather improvement of [data retention](https://en.wikipedia.org/wiki/Data_retention) management in EdgeX (which could involve Core Data, Scheduler, device metadata, etc).

Reply:
Agree - I have renamed accordingly and pulled mention of "cache" as it relates to implementation thoughts.